views/update/update-interrupt-card.html

```diff
@@ -15,23 +15,22 @@
   <div class="govuk-grid-column-two-thirds">
     <div class="interrupt-card">
       <h1 class="govuk-heading-l">Before you start</h1>
-      <h2 class="govuk-heading-m">Agent assurance codes for verification checks</h2>
+      <h2 class="govuk-heading-m">Authentication code</h2>
       <p class="govuk-body">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. 
-        Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. 
-        Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum
+      You will need the overseas entity's authentication code to file this update statement.
       </p>
-      <h2 class="govuk-heading-m">Trusts</h2>
       <p class="govuk-body">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed ex felis, finibus ut luctus vitae, vulputate in sem. 
-        Mauris tincidunt erat non diam condimentum, id lacinia odio scelerisque. 
-        Praesent dapibus augue sit amet lorem ullamcorper, eget condimentum enim elementum. 
-        Nullam scelerisque, libero eget ultrices aliquet, turpis velit ullamcorper urna, id dapibus lacus velit id elit.
+      If you need to request one, search for the overseas entity on the <a href="https://find-and-update.company-information.service.gov.uk/" class="govuk-link">Companies House register</a>, then select 'Request an authentication code'. We'll send the code to the email address that we have on record for the overseas entity.
```

Comment:
This link will not work in the lower envs tho ? please check where it leads to in reserve1 
Not sure if this is a requirement for the link to be env sensitive ? pls check with Royston/Jeremy

Reply:
Had a chat with Royston -- this link is fine as it's intended for the live release so shouldn't impact testing.